client/src/App.js

```diff
@@ -1,11 +1,69 @@
+import AddVideo from "./componets/ADDVIDEO/AddVideo";
+import SingleVideo from "./componets/SINGLEVIDEO/SingelVideo";
+import Search from "./componets/Search/Search";
+import Data from "./VideoData.json";
 import "./App.css";
+import Row from "react-bootstrap/Row";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import { useState, useEffect } from "react";
 
 function App() {
+  const [videos, setVideos] = useState(Data);
+  const [videosFilter, setVideosFilter] = useState("");
```

Comment:
What does `videoFilter` represent?

Reply:
Can you think of a slightly clearer variable name?

Reply:
I have amended with input and setInput